packages/@vue/cli-plugin-e2e-webdriverio/generator/index.js

```diff
@@ -0,0 +1,33 @@
+const { installedBrowsers } = require('@vue/cli-shared-utils')
+
+module.exports = (api, { webdrivers }) => {
```

Comment:
Need to copy-paste the `prompts.js` from the nightwatch plugin, too.

Reply:
It seems that the prompt actually comes from `packages/@vue/cli/lib/promptModules/e2e.js`. Changing the `prompt.js` in the plugin package has no effect.

Reply:
`promptModules/*` are used on `vue create`, `prompt.js` is for `vue add` and `vue invoke`.